src/components/Jars.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import { useSettings } from '../context/SettingsContext'
+import styles from './Jars.module.css'
+import Sprite from './Sprite'
+import Balance from './Balance'
+
+const Jar = ({ accountIndex, balance, fill }) => {
+  const settings = useSettings()
+
+  const jarSymbol = ((fill) => {
+    switch (fill) {
+      case 1:
+        return 'jar-closed-fill-25'
+      case 2:
+        return 'jar-closed-fill-50'
+      case 3:
+        return 'jar-closed-fill-75'
+      default:
+        return 'jar-closed-empty'
+    }
+  })(fill)
+
+  return (
+    <div className="d-flex flex-column align-items-center gap-1">
+      <Sprite symbol={jarSymbol} width="2rem" height="3rem" />
```

Comment:
```suggestion
      <Sprite symbol={jarSymbol} width="32px" height="48px" />
```

I am getting warnings `Unexpected value 2rem parsing width attribute.` and `Unexpected value 3rem parsing height attribute.` in Firefox – not in Chrome though.